Tasks/SonarQubePostTest/SonarQubePostTest.ps1

```diff
@@ -2,6 +2,10 @@ Write-Verbose "Starting SonarQube PostBuild Step"
 
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
 
+Write-Warning "We are transferring ownership of the SonarQube related build tasks to Sonar Source. Please replace these build tasks with the ones provided by Sonar Source. 
```

Comment:
There are aka.ms links in the messge. I'll update them once the final urls become
 available.